### PR TITLE
progressbar: Improve a11y

### PR DESCRIPTION
**Pull request by hcodes**

### CODE REVIEW

File: common.blocks/progressbar/progressbar.bh.js [hunk 1]
```diff
@@ -1,10 +1,18 @@
 module.exports = function(bh) {
 
     bh.match('progressbar', function(ctx, json) {
-        var val = json.val;
+        var val = json.val || 0;
 
         ctx
-            .js({ val : val })
+            .js({ val : json.val })
```
Comment (sipayRT) [inline common.blocks/progressbar/progressbar.bh.js:7]: `json.val` --> `val`

Comment (hcodes) [inline common.blocks/progressbar/progressbar.bh.js:7]: 0 need for the attr and inline style.

Comment (sipayRT) [inline common.blocks/progressbar/progressbar.bh.js:7]: in this case you will get an error when using getVal(), if BEMJSON hasn't `val`.

Comment (narqo) [inline common.blocks/progressbar/progressbar.bh.js:7]: :+1: for @sipayRT 

It's strange to have default 0 value in templates, while allow JS to fail.

File: common.blocks/progressbar/progressbar.bemhtml [hunk 2]
```diff
@@ -4,11 +4,22 @@ block('progressbar')(
...
     }),
 
+    attrs()(function() {
+        return {
+            role : 'progressbar',
+            'aria-label' : this.ctx.ariaLabel,
+            'aria-labelledby' : this.ctx.ariaLabelledBy,
+            'aria-valuemin' : 0,
+            'aria-valuemax' : 100,
+            'aria-valuenow' : this.ctx.val || 0
```
Comment (narqo) [inline common.blocks/progressbar/progressbar.bemhtml:14]: According to our docs `this.ctx.val` should always [be `Number`](https://en.bem.info/libs/bem-components/v2.0.0/desktop/progressbar/docs/#val). In other cases 

1) you should fix js-mode above or block's JS will fail as @sipayRT have mentioned
2) I insist on adding tests (specs and tmpl-specs) for such cases.

**Pull request closed without merging**